src/LightSpeed/userFunction.js

```diff
@@ -134,16 +134,28 @@ var LSUserFunction = GObject.registerClass({
         this._bindModel(this._global.getModel(this._currentLevel));
     }
 
+    unbindGlobalModel() {
+        this._unbindLevelModel();
+        if (this._global && this._globalNotifyHandler) {
+            this._global.disconnect(this._globalNotifyHandler);
+            this._globalNotifyHandler = null;
```

Comment:
While we're touching this code, it would be nice to be consistent about either setting `this._global = null;` or not when it's unbound.